.npmrc

```diff
@@ -0,0 +1 @@
+registry = "https://registry.npmjs.com/"
```

Comment:
Why add this file? (I saw it in the other PRs as well.) Also, in coffee-lex the .npmrc points to registry.yarnpkg.com (which just proxies to the npm registry, right?), so maybe you set one of them wrong?